account_tax_expense_include/hr_expense_expense.py

```diff
@@ -0,0 +1,127 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    Copyright (C) 2010 - 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm
+
+
+class hr_expense_expense(orm.Model):
+    _inherit = 'hr.expense.expense'
+
+    def move_line_get(self, cr, uid, expense_id, context=None):
+        res = []
+        tax_obj = self.pool.get('account.tax')
+        cur_obj = self.pool.get('res.currency')
+        if context is None:
+            context = {}
+        exp = self.browse(cr, uid, expense_id, context=context)
+        company_currency = exp.company_id.currency_id.id
+
+        for line in exp.line_ids:
+            mres = self.move_line_get_item(cr, uid, line, context)
+            if not mres:
+                continue
+            res.append(mres)
+            tax_code_found = False
+
+            # Calculate tax according to default tax on product
+            taxes = []
+            # Taken from product_id_onchange in account.invoice
+            if line.product_id:
+                fpos_obj = self.pool.get('account.fiscal.position')
+
+                product = line.product_id
+                taxes = product.supplier_taxes_id
+                fpos = line.partner_id.property_account_position
+
+                # If taxes are not related to the product, maybe they are in
+                # the account
+                if not taxes:
+                    expense_account = product.property_account_expense
+                    a = expense_account and expense_account.id
```

Comment:
Could you use a more descriptive variable name. It would make the code more readable.